101-securitycenter-create-automation-for-all-recommendations/azuredeploy.json

```diff
@@ -0,0 +1,91 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"automationName": {
+		  "type": "string",
+		  "minLength": 3,
+		  "maxLength": 24
+		},
+		"location": {
+		  "type": "string",
+		  "defaultValue": "[resourceGroup().location]",
+		  "metadata": {
+			"description": "Location for the automation"
+		  }
+		},
+		"logicAppName": {
+			"type": "string",
+			"minLength": 3
+		},
+		"subscriptionAzureResourceId": {
+			"type": "string",
+			"defaultValue": "[subscription().id]",
+			"metadata": {
+				"description": "The Azure resource id of the subscription"
+			}
+		},
+		"subscriptionId": {
+			"type": "string",
+			"defaultValue": "[subscription().subscriptionId]",
+			"metadata": {
+				"description": "The Azure resource GUID id of the subscription"
+			}
+		},
+		"assessmentSettings": {
+			"type":"object",
+			"metadata": {
+				"description": "The object used for deploying the automation"
+			}
+		}
+	  },
+	"variables": {
+		"descriptionDescription": "scope for subscription {0}",
+		"automationDescription": "automation description for subscription {0}"
+	},
+	"resources": [
+		{
+			"apiVersion": "2019-01-01-preview",
+			"location":"[parameters('location')]",
+			"name":"[parameters('automationName')]",
+			"type":"Microsoft.Security/automations",
+			"properties":{
+				"description":"[variables('automationDescription')]",
+				"isEnabled":true,
+					"actions":[{
+						"actionType":"LogicApp",
+						"logicAppResourceId":"[resourceId('Microsoft.Logic/workflows', parameters('logicAppName'))]",
```

Comment:
you'll need to pass in the resourceGroup name of the logic app as well and use it in the resourceId functions.

Reply:
Correct.  :)